mkosi.md

```diff
@@ -127,119 +127,6 @@ The following command line verbs are known:
 : This verb is equivalent to the `--help` switch documented below: it
   shows a brief usage explanation.
 
-## Execution flow
```

Comment:
Even with a single phase the flow diagram is still helpful, so that users know what happens at which point.